src/fundus/publishers/fr/__init__.py

```diff
@@ -0,0 +1,13 @@
+from fundus.publishers.base_objects import PublisherEnum, PublisherSpec
+from fundus.publishers.fr.le_monde import LeMondeParser
+from fundus.scraping.filter import inverse, regex_filter
+from fundus.scraping.html import NewsMap, Sitemap
+
+
+class FR(PublisherEnum):
+    LeMonde = PublisherSpec(
+        name="Le Monde",
+        domain="https://www.lemonde.fr/",
+        sources=[NewsMap("https://www.lemonde.fr/sitemap_news.xml", reverse=True)],
```

Comment:
```suggestion
        sources=[NewsMap("https://www.lemonde.fr/sitemap_news.xml")],
```
It seems that this one is already in order from recent to older.